src/components/grapher/Dot.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <g class="dot--dot-container">
+    <circle
+      class="dot--circle"
+      :r="radius"
+      data-test="dot--circle"
+      :stroke="dotAppearance.color"
+      stroke-width=".3"
+      :fill="dotAppearance.fill"
+      :fill-opacity="dotAppearance.filled ? 1 : 0"
+    />
+    <line
+      v-if="dotAppearance.fwSlash"
```

Comment:
The way you build up an appearance by composition here is pretty neat. I guess an X shape would just be one for which both `fwSlash` and `bwSlash` returns true, eh?

Reply:
Yep exactly! Credit for thinking about this goes to Brandon: https://github.com/calband/calchart-redesign-old/blob/master/src/grapher/GrapherDot.vue